second-edition/src/ch06-01-defining-an-enum.md

```diff
@@ -194,21 +214,28 @@ enum Message {
 }
 ```
 
-<span class="caption">Listing 6-2: A `Message` enum whose variants each store
-different amounts and types of values</span>
+<span class="caption">Listado 6-2: Una enumeración de `Message` cuyas
+    variantes almacene cada una diferentes cantidades y tipos de valores</span>
 
-This enum has four variants with different types:
+Esta enumeración tiene cuatro variantes con diferentes tipos:
 
+
+* `Quit` no tiene datos asociados con el del todo.
+* `Move` incluye una estructura anónima dentro de ella.
+* `Write` incluye un solo `String`.
+* `ChangeColor` incluye tres `i32`.
+=======
 * `Quit` has no data associated with it at all.
```

Comment:
This part seems duplicated as well

Reply:
@thecodix 
How can I solve that? I can not delete the text in English .. please help me..